src/serac/physics/utilities/variational_form/detail/hexahedron_h1.inl

```diff
@@ -0,0 +1,72 @@
+template <int p, int c>

```

Comment:
These `inl` files need some brief comments for what they are computing at a high level. E.g. "This file computes shape functions and gradients for tensor product Lagrange basis functions with Gauss-Lobatto quadrature on hexahedral elements."

Reply:
I added a description at the `finite_element` class template declaration of the concept each specialization should satisfy. I'll add some additional clarification on the individual specializations as well.